features/admin.applications.v1/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -968,6 +968,37 @@ export const AttributeSettings: FunctionComponent<AttributeSettingsPropsInterfac
         }
     });
 
+
+    /**
+     *  Get the final value of includeUserDomain
+     */
+    const getIncludeUserDomainFinalValue = ((advanceSettingValues : AdvanceSettingsSubmissionInterface) => {
+
+        let includeUserDomain: boolean = advanceSettingValues?.subject.includeUserDomain;
+
+        if (onlyOIDCConfigured && (typeof advanceSettingValues?.subject?.claim === "string" &&
+        advanceSettingValues?.subject?.claim === DefaultSubjectAttribute)) {
+            includeUserDomain = false;
+        }
```

Comment:
No need to nest conditions because only && is used.

```suggestion
        if (
            onlyOIDCConfigured
            && typeof advanceSettingValues?.subject?.claim === "string"
            && advanceSettingValues?.subject?.claim === DefaultSubjectAttribute
        ) {
            includeUserDomain = false;
        }
```